src/Common/src/Interop/Unix/System.Native/Interop.MountPoints.FormatInfo.cs

```diff
@@ -130,7 +178,7 @@ private static int GetFormatInfoForMountPoint(string name, out string format, ou
         /// <returns>The recognized drive type.</returns>
         private static DriveType GetDriveType(string fileSystemName)
         {
-            // This list is based primarily on "man fs", "man mount", "mntent.h", "/proc/filesystems",
+            // This list is based primarily on "man fs", "man mount", "mntent.h", "/proc/filesystems", coreutils "stat.c",
```

Comment:
Is this meant to be the stat.c from coreutils?  If so, it should probably be:
```
coreutils` "stat.c"
```
rather than
```
coreutils "stat.c"
```
If that's not the case, then it seems like there's a comma missing.